gui/wxpython/gmodeler/model.py

```diff
@@ -695,7 +695,7 @@ def Run(self, log, onDone, parent=None):
                     parent=parent,
                     message=_("Variables below not defined:")
                     + "\n\n"
-                    + "\n".join(map(lambda x: "%s: %s (%s)" % (x[0], x[1], x[2]), err)),
+                    + "\n".join(("%s: %s (%s)" % (x[0], x[1], x[2]) for x in err)),
```

Comment:
Should not this be solved with an f-string?

Reply:
```suggestion
                    + "\n".join(f"{x[0]}: {x[1]} ({x[2]})" for x in err),
```